pkgs/build-support/setup-hooks/move-systemd-user-units.sh

```diff
@@ -0,0 +1,24 @@
+#!/run/current-system/sw/bin/bash
+
+# This setup hook, for each output, moves everything in
+# $output/lib/systemd/user to $output/share/systemd/user, and replaces
+# $output/lib/systemd/user with a symlink to
+# $output/share/systemd/user.
+
+fixupOutputHooks+=(_moveSystemdUserUnits)
+
+_moveSystemdUserUnits() {
+    if [ "${dontMoveSystemdUserUnits:-0}" = 1 ]; then return; fi
+    if [ ! -e "${prefix:?}/lib/systemd/user" ]; then return; fi
+    local source="$prefix/lib/systemd/user"
+    local target="$prefix/share/systemd/user"
+    echo "moving $source/* to $target"
```

Comment:
Is the glob here supposed to catch?
If so the `*` needs to be unquoted.

Reply:
The intent is to print `*`, not expand to file names. Like the `move-sbin.sh` hook.